admin/broadleaf-open-admin-platform/src/main/resources/messages/GeneratedMessagesEntityFramework.properties

```diff
@@ -525,6 +526,7 @@ OfferItemRestrictionRuleType_tooltip=<ul><li><b>None</b> - Item cannot participa
 OfferImplRelatedTargetQualifier_tooltip=In order to receive this offer, qualifying and target items must be dependent on each other.<br /> For example: Purchase item X with dependent order item Y, and get a discount off. Offer will not apply if purchasing either X or Y separately.
 OfferImplMaxUsesPerOrder_tooltip=Specifies how many times this offer can be used on a single order.<br /> "<b>0</b>" indicates unlimited use.
 OfferImplMaxUsesPerCustomer_tooltip=Specifies how many times this offer can be used by a single customer. By default, this condition is only enforced for <b>registered</b> customers.<br /> A value of "<b>0</b>", or leaving the field blank, indicates unlimited use.
+OfferImplMinimumDaysPerUsage_tooltip=Specifies the minimum period (in days) this offer can be used along with "Max Uses per Customer" before resetting. By default, this condition is only enforced for <b>registered</b> customers.<br /> A value of "<b>0</b>", or leaving the field blank, indicates unlimited use.
```

Comment:
Used in conjunction with "Max Uses per Customer".   This property indicates the number of days in which the max uses per customer value is enforced.    Leaving the field blank or < 1 has no effect.